enterprise/models.py

```diff
@@ -2888,7 +2888,7 @@ def get_context(self):
         if self.enterprise_customer:
             return [str(self.enterprise_customer.uuid)]
 
-        return super().get_context()
```

Comment:
https://github.com/openedx/edx-rbac/blob/f4d22abe35bdc3cff82f324167806d924fd95ef1/edx_rbac/models.py#L83

^ the default returns None, and it was overridden in the Mixin on line 2813. Should we instead remove the mixin override?

I guess, do we also want to return none for EnterpriseFeatureUserRoleAssignment as well?